standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -2101,7 +2102,7 @@ service ThriftHiveMetastore extends fb303.FacebookService
                        throws(1:MetaException o1, 2:NoSuchObjectException o2)
 
   // get partitions give a list of partition names
-  list<Partition> get_partitions_by_names(1:string db_name 2:string tbl_name 3:list<string> names)
+  list<Partition> get_partitions_by_names(1:string db_name 2:string tbl_name 3:list<string> names, 4:bool get_col_stats)
```

Comment:
Do you suggest that we add a new API get_partitions_by_names which accepts a new structure GetPartitionsByNamesRequest as an argument. Also retain the old API get_partitions_by_names() instead of changing it?

Reply:
yes

Reply:
Done.